service-base/src/main/java/org/eclipse/hono/service/management/device/EventBusDeviceManagementAdapter.java

```diff
@@ -1,213 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2016, 2019 Contributors to the Eclipse Foundation
- *
- * See the NOTICE file(s) distributed with this work for additional
- * information regarding copyright ownership.
- *
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License 2.0 which is available at
- * http://www.eclipse.org/legal/epl-2.0
- *
- * SPDX-License-Identifier: EPL-2.0
- *******************************************************************************/
-
-package org.eclipse.hono.service.management.device;
-
-import java.net.HttpURLConnection;
-import java.util.Objects;
-import java.util.Optional;
-
-import org.eclipse.hono.client.ClientErrorException;
-import org.eclipse.hono.service.EventBusService;
-import org.eclipse.hono.service.management.Id;
-import org.eclipse.hono.service.management.OperationResult;
-import org.eclipse.hono.service.management.Result;
-import org.eclipse.hono.service.management.Util;
-import org.eclipse.hono.util.EventBusMessage;
-import org.eclipse.hono.util.RegistryManagementConstants;
-
-import io.opentracing.Span;
-import io.opentracing.SpanContext;
-import io.vertx.core.Future;
-import io.vertx.core.Promise;
-import io.vertx.core.Verticle;
-import io.vertx.core.json.JsonObject;
-
-/**
- * An adapter, hooking up the {@link DeviceManagementService} with the event bus.
- * <p>
- * In particular, this base class provides support for receiving service invocation request messages via vert.x' event
- * bus and route them to specific methods corresponding to the operation indicated in the message.
- */
-public abstract class EventBusDeviceManagementAdapter extends EventBusService
```

Comment:
FMPOV the service-base module contains public API, doesn't it? If so, then we should first deprecate this class, indicating that it is no longer used/necessary and that it will be removed in Hono 2.0.0. WDYT?